### ISSUE TITLE
Ruby has a runtime dependency on another ruby

### ISSUE THREAD

**Issue by edolstra** [NixOS/nixpkgs#5914]
For example:
```
$ nix-store -qR $(nix-build -A ruby) | grep ruby
/nix/store/z2b4cvsy9j97bd0dqqy96gsnml4zr6q9-ruby-1.9.3-p547
/nix/store/n5rp0d6njynn7n7mnvi40fvip4203chv-ruby-1.9.3-p547
```
This makes Ruby's closure about 80 MB larger than it needs to be.

The reason is the file `lib/ruby/1.9.1/x86_64-linux/rbconfig.rb`, which contains two references to the "base ruby".

@cstrahan

**Comment by copumpkin** [comment]
Seems like an instance of the much larger problem I describe in #5895

**Comment by cstrahan** [comment]
Ugh. Thanks for the heads up.

**Comment by cstrahan** [comment]
Fixed in ad026dab9808eb6917a78703ebb20475bd122a7d.
